pyproject.toml

```
@@ -25,7 +25,7 @@ python = "^3.7"
 numpy = "^1.20"
 scipy = "^1.6.1"
 lark = "^0.11.1"
-rpcq = "^3.6.0"
+rpcq = "^3.9.0"
```

```suggestion
rpcq = "^3.9.2"
```
Should this be required as at least 3.9.2 to avoid the multiprocess issue?

---

Forgot to push. Thanks :)